iib/workers/tasks/build.py

```diff
@@ -755,14 +752,24 @@ def handle_add_request(
                     ' '.join(excluded_bundles),
                 )
 
-        _opm_index_add(
-            temp_dir,
-            resolved_bundles,
-            prebuild_info['binary_image_resolved'],
-            from_index_resolved,
-            overwrite_from_index_token,
-            (prebuild_info['distribution_scope'] in ['dev', 'stage']),
-        )
+        if not is_fbc:
```

Comment:
Optional: Can we please do `if is_fbc` and `else` instead?

Reply:
Sure why not, I guess it is more readable that way, thanks.